t51-projects/gh-project-board.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+
+function curl_fetch() {
+
+	$url = 'https://api.github.com/users/jessLundie'; // Set source URL
+
+	$column_id = "15445284"; // Column id, found in the column link here: https://d.pr/i/W7RQJB
+
+	$project_url = "https://api.github.com/projects/columns/" . $column_id . "/cards"; // Set project URL
+
+	$token = "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"; // Basic Auth Token
```

Comment:
head's up, you _never_ want to commit actual auth tokens. it would be best to login to Github and deactivate this one as it's not published to the world 😉